src/test/java/edu/byu/cs240/checkstyle/readability/CommentedCodeTest.java

```diff
@@ -53,6 +53,14 @@ public void should_FindNoErrors_when_TextBlock() throws CheckstyleException {
     }
 
 
+    @Test
+    @DisplayName("Should find no errors in code that has '/*' or related character sequences in a string")
+    public void should_FindNoErrors_when_commentInString() throws CheckstyleException {
+        String fileName = "testInputs/commentedCode/should_FindNoErrors_when_commentInString.java";
```

Comment:
I love that you led with well crafted tests to verify that the code actually behaves as it should. Nice work!